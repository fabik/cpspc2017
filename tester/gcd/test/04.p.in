12
1 3 6 10 11 12 15 16 18 19 22 26
22 26 26 