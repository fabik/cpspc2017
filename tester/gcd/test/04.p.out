1
27
49
68
86
102
117
129
140
150
156
159
